### PR TITLE
Tracking inventory

### PR DESCRIPTION
**Pull request by amymhaddad**
Adding track_inventory project for PR review. This project gets inventory information from user and creates a tabular table.

### CODE REVIEW

**Comment by amymhaddad** [comment]
@paulghaddad This project is updated. Pls review these files:
main.py
create_inventory_tracking_report.py 
inventory_item.py
inventory_tracking_management.py
inventory_report.py

File: track_inventory/create_inventory_tracking_report.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
+"""Get user input and create an inventory report"""
```
Comment (paulghaddad) [inline track_inventory/create_inventory_tracking_report.py:1]: Nice module name.

File: track_inventory/create_inventory_tracking_report.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+
+from prettytable import from_csv
+from inventory_report import InventoryReport
+
+
+def get_inventory_report_name():
```
Comment (paulghaddad) [inline track_inventory/create_inventory_tracking_report.py:8]: Good function.

File: track_inventory/inventory_tracking_management.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
+"""Write report inventory to a json or csv file"""
```
Comment (paulghaddad) [inline track_inventory/inventory_tracking_management.py:1]: Excellent; nice functions here and this module only contains items related to outputting to a file.

File: track_inventory/main.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+import sys
+from inventory_tracking_management import write_inventory_to_csv
+
+from create_inventory_tracking_report import (
+    get_inventory_report_details,
+    csv_tablular_report,
+)
+
+
+def main(filename):
```
Comment (paulghaddad) [inline track_inventory/main.py:12]: Nice work. This `main` function is well-organized into input -> core -> output.

File: track_inventory/inventory_report.py [hunk 5]
```diff
@@ -0,0 +1,23 @@
+"""Template to store inventory report data """
+from inventory_item import InventoryItem
+
+
+class InventoryReport:
```
Comment (paulghaddad) [inline track_inventory/inventory_report.py:5]: Perfect class all around!

File: track_inventory/inventory_item.py [hunk 6]
```diff
@@ -0,0 +1,12 @@
+"""Template to create an inventory report"""
+
+
+class InventoryItem:
```
Comment (paulghaddad) [inline track_inventory/inventory_item.py:4]: 👏

**Review by paulghaddad — CHANGES_REQUESTED** [review]
Awesome work! 💯

**Pull request merged**